src/test/java/io/confluent/examples/streams/kafka/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -254,16 +250,8 @@ private TopicsDeletedCondition(final String... topics) {
 
     @Override
     public boolean conditionMet() {
-      final Properties properties = new Properties();
```

Comment:
Should we keep this old code, put it into comments, and add a comment that we should change this back after this bug is fixed?

Reply:
@mjsax I'm not a big fan of leaving commented out code even with an explanation, so I've added a `TODO` saying to use `AdminClient` once the issue is fixed.  Does that work for you?

This is just my personal preference so I'll add the code back in comments if you insist.

Reply:
SGTM -- just referencing the ticket is better.

(Just want to make sure, we don't say "hey, why don't we use `AdminClient` here" and reintroduce this issue.)

Reply:
:+1: This todo is perfect. When we decide to resolve it, we can check `git blame` to find the prior code if we like.